### ISSUE TITLE
Installer does not set EFI partition as ESP

### ISSUE THREAD

**Issue by Teh404Gal2** [Vanilla-OS/Albius#57]
Installed vanilla OS this week and noticed a constant update that wouldnt apply on my laptop (lenovo v15 g2 alc), after trying the cli tool it told me that my ESP partition's bit wasnt set correctly
After going into gnome disks i manually changed the efi from "data base" to "ESP system", after a reboot that worked, and also i no longer had random times when the bios couldnt detect the efi (i thought i was a bios bug, but no

**Comment by taukakao** [comment]
Do you mean "EFI System" in gnome disks?

**Comment by Teh404Gal2** [comment]
Yeah, i think thats it, im using french local so the options where "Base de données" -> "Système EFI" so i transliterated it

**Comment by taukakao** [comment]
Ok, thanks. Should be an easy enough fix.
